test/unit/behaviors.html

```diff
@@ -127,6 +127,27 @@
     assert.equal(el.attributes.user.value, 'user', 'Behavior hostAttribute overrode user attribute');
   });
 
+  test('behaviour is null generates warning', function() {
+    var warned = false, oldWarn = Polymer.Base._warn;
+    Polymer.Base._warn = function (message) {
```

Comment:
I indeed tried to do this, but sadly it was not possible. The warning is generated on call-time of `Polymer({})` and therefore there is no reference to `this` yet to override the function. Therefore I had to override the prototype, call `Polymer({})` which generated the warning, and then verify that upon creating this warning was generated.